docs/reference/datatiers.asciidoc

```diff
@@ -37,9 +37,9 @@ TIP: The performance of an {es} node is often limited by the performance of the
 For example hardware profiles, refer to Elastic Cloud's {cloud}/ec-reference-hardware.html[instance configurations]. 
 Review our recommendations for optimizing your storage for <<indexing-use-faster-hardware,indexing>> and <<search-use-faster-hardware,search>>.
 
-IMPORTANT: {es} generally expects nodes within a data tier to share the same 
-hardware profile. Variations not following this recommendation should be 
-carefully architected to avoid <<hotspotting,hot spotting>>.
+IMPORTANT: {es} generally expects nodes within a data tier to share the same hardware profile (such as CPU, RAM, disk capacity) 
```

Comment:
```suggestion
IMPORTANT: By default, {es} assumes nodes within a data tier share the same hardware profile (such as CPU, RAM, disk capacity) 
```
Should we potentially take out "generally" and/or add "by default" in order to firm this as baseline expectation? Or go as far to say ES "assumes" this is true so if you want to do different you need to "carefully architect" (I like that phrasing)

Reply:
I'd just say "Elasticsearch expects"